src/components/AssetHeader/AssetMarketData.tsx

```diff
@@ -128,11 +128,7 @@ export const AssetMarketData: React.FC<AssetMarketDataProps> = ({ assetId }) =>
                 <Text translation='assets.assetDetails.assetHeader.availableSupply' />
               </StatLabel>
               <StatValue isLoaded={isLoaded}>
-                <Amount
```

Comment:
Tested with this expensive bad boi and noticed it zero'd out its supply - this makes things better

<img width="448" alt="image" src="https://github.com/user-attachments/assets/36110e73-950b-49ab-8981-555b85505438">
<img width="928" alt="image" src="https://github.com/user-attachments/assets/f52231b2-5d5a-47a4-b585-d00799583f9d">